site/src/examples/wealth-and-health-of-nations/index.css

```diff
@@ -0,0 +1,41 @@
+#wealth-and-health-of-nations {
+  margin: 20px auto;
+  width: 100%;
+  height: 400px;
+}
+#wealth-and-health-of-nations > .cartesian-chart,
+.plot-area-container > .plot-area {
+  overflow: visible;
```

Comment:
I hadn't realised the data breached the x domain, that seems like a pretty specialist use-case so happy with that one. I think you can get rid of the first rule using the margin settings through e.g. -
```js
.margin({top: 1, right: 20, bottom: 20, left: 30})
```